src/overlays/actors/ovl_En_Tanron3/z_en_tanron3.c

```diff
@@ -28,39 +36,380 @@ const ActorInit En_Tanron3_InitVars = {
     (ActorFunc)EnTanron3_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BB9750 = {
-    { COLTYPE_HIT3, AT_ON | AT_TYPE_ENEMY, AC_ON | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK3, { 0xF7CFFFFF, 0x00, 0x02 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_HIT3,
+        AT_ON | AT_TYPE_ENEMY,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK3,
+        { 0xF7CFFFFF, 0x00, 0x02 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 7, 10, -5, { 0, 0, 0 } },
 };
 
-#endif
+// This actor has two colliders (one for AC and one for AT), but uses the same
+// ColliderCylinderInit for both of them, leaving this one totally unused.
+static ColliderCylinderInit sUnusedCylinderInit = {
+    {
+        COLTYPE_HIT3,
+        AT_ON | AT_TYPE_ENEMY,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK3,
+        { 0xF7CFFFFF, 0x00, 0x02 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
+    { 20, 20, -10, { 0, 0, 0 } },
+};
+
+extern FlexSkeletonHeader D_0600DA20;
+extern AnimationHeader D_0600DAAC;
+
+void EnTanron3_CreateEffect(GlobalContext* globalCtx, Vec3f* effectPos) {
+    UnkTanron3Effect* effectPtr = (UnkTanron3Effect*)globalCtx->specialEffects;
+    s16 i;
+
+    for (i = 0; i < 150; i++, effectPtr++) {
+        if ((effectPtr->unk_00 == 0) || (effectPtr->unk_00 == 1)) {
+            effectPtr->unk_00 = 2;
+            effectPtr->unk_04 = *effectPos;
+            effectPtr->unk_10 = *sZeroVec;
+            effectPtr->unk_1C = *sZeroVec;
+            effectPtr->unk_1C.y = -2.0f;
+            effectPtr->unk_34.x = 0.1f;
+            effectPtr->unk_34.y = 0.0f;
+            effectPtr->unk_34.z = Rand_ZeroFloat(M_PI * 2);
+            effectPtr->unk_02 = Rand_ZeroFloat(100.0f);
+            effectPtr->unk_10.x = randPlusMinusPoint5Scaled(25.0f);
+            effectPtr->unk_10.z = randPlusMinusPoint5Scaled(25.0f);
+            return;
```

Comment:
```suggestion
            break;
```
since it's more breaking out of the loop than escaping the function.